src/mlpack/bindings/R/mlpack/cleanup

```diff
@@ -0,0 +1,5 @@
+## compilation and editing objects
+rm -f src/*.o src/*.so src/*.dylib src/*~ *~
+
+## autoconf/configure leftovers
+rm -rf autom4te.cache/ config.log config.status src/Makevars confdefs.h
```

Comment:
Where will this file be used?  I have a concern that it isn't platform-independent, but I'm not sure what it's used for, so maybe my concern doesn't matter.

Reply:
A common (yet optional) part of R packages.  If present, script `cleanup` can be run at end.  Maybe have gotten copied over into MLPACK proper by association :)

Reply:
Aye, it was copied over when the shell of the _R_ package was incorporated into the binding, c.f. 

https://github.com/Yashwants19/mlpack/tree/8a46560c62575a5ec3eec5bedb5915ceb95f3041/src/mlpack/bindings/R/mlpack